### PR TITLE
Vercel deployment

### PR DESCRIPTION
**Pull request by noahwaldner**

### CODE REVIEW

File: packages/blog-with-markdown-files/components/templates/BlogHomeTemplate.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { BlogPost } from 'types/blog'
+
+type BlogPostPageProps = {
+	blogPost: BlogPost
+}
+
+export default function BlogHomeTemplate({ blogPost }: BlogPostPageProps): React.ReactElement {
+	return (
+		<div>
+			<h1>{blogPost.title}</h1>
```
Comment (lorenzogm) [inline packages/blog-with-markdown-files/components/templates/BlogHomeTemplate.tsx:10]: We can use Typography here too :)

**Comment by noahwaldner** [comment]
I added dummy components for BlogHomeTemplate and Author Widgets as empty modules are causing errors.

File: packages/components/organisms/AuthorWidget.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      <AuthorImage />
+
+      <Typography variant="h1">{authorName}</Typography>
+      <Typography variant="p">{content}</Typography>
+    </Container>
+  )
+}
+
+
+import React from 'react'
```
Comment (lorenzogm) [inline packages/components/organisms/AuthorWidget.tsx:19]: I think you need a new file

**Review by lorenzogm — CHANGES_REQUESTED** [review]
There are files non related with the vercel deployment that maybe you want to take out

**Comment by noahwaldner** [comment]
I didn't remove the files completely, as this would result in other files needing a change too. I wanna avoid doing too many changes which need to be reverted once we implement these components

**Review by SgiobairOg — APPROVED** [review]

**Comment by SgiobairOg** [comment]
I think we're okay to merge now @lorenzogm. Agree?

**Comment by lorenzogm** [comment]
Perfect now, at least now we see they are dummy components!

**Review by lorenzogm — APPROVED** [review]

**Pull request merged**